doc/genomedata.rst

```diff
@@ -374,6 +374,8 @@ format and the bigWig file format:
 
 - There is only one track per bigWig file and it is implicitly set to the
   filename of the bigWig.
+- Any track indicies or (file)name specfied are only considered for shaping
```

Comment:
can you please avoid this use of parentheses, be more explicit about the different possibilities